### PR TITLE
[KZL-1046] Add Kuzzle Nightly & Redis 3 and 4 test

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
## What does this PR do ?
Update our Travis configuration to launch heavy and time consuming tests by night using Travis CRON.
  
### How should this be manually tested?
Look at Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -170,13 +170,69 @@ jobs:
...
+      services: docker
+
+      cache:
+        directories:
+          - $HOME/.pm2 
+
+      env:
+        - DOCKER_COMPOSE_VERSION=1.9.0
+        - NODE_LTS=8
+        - REDIS_VERSION=latest
```
Comment (benoitvidis) [inline .travis.yml]: Shouldn't we rather stick to a tag (can be set to `5` imo to follow last versions)?

Comment (alexandrebouthinon) [reply .travis.yml]: Yes you're right :slightly_smiling_face:

**Comment by alexandrebouthinon** [comment]
@Aschen: I update the Redis version in all repo Docker Compose files. I'll update other docker-compose soon

File: .ci/test-aarch64.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ services:
       - TRAVIS_REPO_SLUG
 
   redis:
-    image: redis:3.2
+    image: redis:${REDIS_VERSION:-5}
```
Comment (scottinet) [inline .ci/test-aarch64.yml]: Why do we default to redis 5 if we're only testing v3 and v4?

Comment (alexandrebouthinon) [reply .ci/test-aarch64.yml]: Redis 5 is tested in `Integration tests` > `Node v8`. Because this is the new default version.

**Pull request merged**